### PR TITLE
Update CF with promo dynamodb read permissions

### PR DESCRIPTION
**Pull request by tomverran**

### CODE REVIEW

File: cloud-formation/subscriptions-app.cf.json [hunk 1]
```diff
@@ -205,7 +232,8 @@
                 "ConnectionDrainingPolicy": {
                     "Enabled" : "true",
                     "Timeout" : "60"
-                }
+                },
+                "CrossZone": "true"
```
Comment (paulbrown1982) [inline cloud-formation/subscriptions-app.cf.json]: As we're about to do a CloudFormation change, I've drive-by'd this very small tweak (which we use in many other places) to enable cross-zone routing from the ELB, which is recommended in the Trusted Advisor.

**Comment by tomverran** [comment]
This is what is live now so I am going to go ahead and merge it

**Comment by mario-galic** [comment]
Fail-in-PROD was due to Travis CI build problem.
